sdk/identity/Azure.Identity/src/AadIdentityClient.cs

```diff
@@ -17,130 +17,51 @@ namespace Azure.Identity
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
+            return SendAuthRequest(request, cancellationToken);
         }
 
         public virtual async Task<AccessToken> AuthenticateAsync(string tenantId, string clientId, X509Certificate2 clientCertificate, string[] scopes, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope("Azure.Identity.AadIdentityClient.Authenticate");
-            scope.Start();
+            using Request request = CreateClientCertificateAuthRequest(tenantId, clientId, clientCertificate, scopes);
 
-            try
-            {
-                using Request request = CreateClientCertificateAuthRequest(tenantId, clientId, clientCertificate, scopes);
-                try
-                {
-                    return await SendAuthRequestAsync(request, cancellationToken).ConfigureAwait(false);
-                }
-                catch (RequestFailedException ex)
-                {
-                    throw new AuthenticationFailedException(AuthenticationRequestFailedError, ex);
-                }
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
+            return await SendAuthRequestAsync(request, cancellationToken).ConfigureAwait(false);
         }
 
         public virtual AccessToken Authenticate(string tenantId, string clientId, X509Certificate2 clientCertificate, string[] scopes, CancellationToken cancellationToken = default)
         {
-            using DiagnosticScope scope = _clientDiagnostics.CreateScope("Azure.Identity.AadIdentityClient.Authenticate");
-            scope.Start();
+            using Request request = CreateClientCertificateAuthRequest(tenantId, clientId, clientCertificate, scopes);
 
-            try
-            {
-                using Request request = CreateClientCertificateAuthRequest(tenantId, clientId, clientCertificate, scopes);
-                try
-                {
-                    return SendAuthRequest(request, cancellationToken);
-                }
-                catch (RequestFailedException ex)
-                {
-                    throw new AuthenticationFailedException(AuthenticationRequestFailedError, ex);
-                }
-            }
-            catch (Exception e)
-            {
-                scope.Failed(e);
-                throw;
-            }
+            return SendAuthRequest(request, cancellationToken);
         }
 
         private async Task<AccessToken> SendAuthRequestAsync(Request request, CancellationToken cancellationToken)
         {
-            Response response = await _pipeline.SendRequestAsync(request, cancellationToken).ConfigureAwait(false);
+            Response response = await Pipeline.Pipeline.SendRequestAsync(request, cancellationToken).ConfigureAwait(false);
```

Comment:
This "Pipeline.Pipeline" looks confusing. I know it's internal code, but maybe you could make the names a bit more descriptive for future maintainers of the code.

Reply:
renamed the property HttpPipeline